zktraffic/endpoints/stats_server.py

```diff
@@ -77,27 +79,50 @@ def wakeup(self):
   def has_stats(self):
     return len(self._get_stats('per_path')) > 0
 
-  def _get_stats(self, name, prefix=''):
+  def _get_stats(self, name, prefix='', output_array=False):
     stats_by_opname = self._stats.stats(name, self._max_results)
 
-    stats = {}
+    if output_array:
+        stats_arr = []
+    else:
+        stats = {}
     for opname, opstats in stats_by_opname.items():
       for path, value in opstats.items():
-        stats["%s%s%s" % (prefix, opname, path)] = value
-
-    return stats
+        if output_array:
+            #TODO: split prefix to each own key
+            #TODO: when per_ip: split path by ':' and add IPs to their own key
+            if prefix.endswith('/'):
+                prefix=prefix[:-1]
```

Comment:
Consider adding an extra space around the equal sign.

Reply:
mixed multiple of those issues